docs/release-notes/february-2023.md

```diff
@@ -0,0 +1,86 @@
+
+## DeepSemgrep → Semgrep Pro Engine
+
+- DeepSemgrep has been bundled with other functionalities to offer you **Semgrep Pro Engine**! Semgrep Pro Engine is fully available for [Team or higher tier](https://semgrep.dev/pricing) users. See [Semgrep Pro](/deepsemgrep/deepsemgrep-introduction/) documentation. 
+- Experimental support for Apex language is now available in Semgrep Pro Engine.
+- The following already **deprecated** flags have been completely removed and substituted:
+    - `-deep` has been removed and substituted by `-pro`.
+    - `-interfile` has been removed and substituted by `-pro`.
+    - `-interproc` has been removed and substituted by `-pro-intrafile`.
+- Removed already **deprecated** command:
+`install-deep-semgrep` has been removed and substituted by `install-semgrep-pro`.
+
+## Semgrep App → Semgrep Cloud Platform
+
+- Semgrep App is now Semgrep Cloud Platform!
+- Group by rule became the default view on the Findings page (now labeled as **Code** page) of Semgrep Cloud Platform. This view enables you to see which rules detected certain findings. You can always switch to the old no grouping view. For more information, see [Grouping by rule](/semgrep-app/findings/#grouping-by-rule).
+    ![Screenshot of the Findings page with findings grouped by rule](/img/app-findings.png)<br />
+- Taint analysis traces are now displayed on the finding detail page, helping you to track tainted data as they propagate through your code. See [Viewing the path of tainted data](/semgrep-app/findings/#viewing-the-path-of-tainted-data) to try out this feature.
+    ![Data flow in Finding details page](/img/cloud-platform-findings-details-data-flow.png)<br />
+
+## Semgrep Supply Chain
+
+- Semgrep Supply Chain now displays a summary of vulnerabilities and scan data in the **Semgrep Cloud Platform** > **Dashboard** page. This enables users to view a report of findings for both their first-party and third-party code.
+- Java is now a Generally Available language in Semgrep Supply Chain.
+- Various fixes and improvements to performance.
+
+## Semgrep Registry
+
+- When you now click on a hyperlink header with the rule name in [Semgrep Registry](https://semgrep.dev/explore), the link opens a new tab with the rule in either the Semgrep Playground (if you are logged out) or in Semgrep Editor (if you are logged in). This can help you to keep the Registry page open but look at or modify the rule.
+
+## Documentation
+
+- Slight improvements to relevancy in Semgrep Docs’s search bar.
+- Updates to **Supported Languages** > **Semgrep Supply Chain** and **Semgrep Code** > **Semgrep Pro Engine**.
+- Updates to **Semgrep Code** > **Alerts and Notifications** to consolidate all methods to send and receive scan data, such as findings.
+- Updates to **Pricing and Billing** to reflect the differences between Semgrep OSS Engine and Semgrep Code.
+- New documentation for Semgrep Code.
```

Comment:
TODO

Reply:
Semgrep Code is an entirely new segment, you can see it on the left sidebar, there's a new doc for "semgrep code"

Reply:
@s-santillan got it. Would you agree with this rephrasing:

```suggestion
- Added new documentation category Semgrep Code.
```